tools/engine_compare.py

```
@@ -159,6 +169,16 @@ def convert_rdocx(rdocx: Path, docx: Path, pdf: Path) -> bool:
     return pdf.exists()
 
 
+def convert_jubarte(jubarte: Path, docx: Path, pdf: Path) -> bool:
+    if is_fresh(pdf, docx):
+        return True
+    pdf.parent.mkdir(parents=True, exist_ok=True)
+    # --force: the CLI refuses to clobber an existing output by default.
+    subprocess.run([str(jubarte), "convert", str(docx), "-o", str(pdf), "--force"],
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)